engine/admin/start-containers-automatically.md

```diff
@@ -0,0 +1,57 @@
+---
+description: How to generate scripts for upstart, systemd, etc.
+keywords: systemd, upstart, supervisor, docker, documentation,  host integration
+redirect_from:
+- /engine/articles/host_integration/
+- /engine/admin/host_integration/
+title: Start containers automatically
+---
+
+Docker provides [restart policies](/engine/reference/run.md#restart-policies---restart)
+to control whether your containers start automatically when they exit, or when
+Docker restarts. Restart policies ensure that linked containers are started in
+the correct order. Docker recommends that you use restart policies, and avoid
+using process managers to start containers.
+
+## Use a restart policy
+
+To configure the restart policy for a container, use the `--restart` flag
+when using the `docker run` command. The value of the `--restart` flag can be
+any of the following:
+
+| Flag             | Description                                                                                     |
+|:-----------------|:------------------------------------------------------------------------------------------------|
+| `no`             | Do not automatically restart the container. (the default)                                       |
+| `on-failure`     | Restart the container if it exits due to an error.                                              |
+| `unless-stopped` | Restart the container unless it is explicitly stopped or Docker itself is stopped or restarted. |
+| `always`         | Always restart the container if it stops.                                                       |
+
+The following example starts a Redis container and configures it to always
+restart unless it is explicitly stopped or Docker is restarted.
+
+```bash
+$ docker run -dit --restart unless-stopped redis
+```
+
+## Use a process manager
```

Comment:
Should we mention the `--live-restore` option in the context of this document? i.e. allow keeping the containers running when restarting the daemon?